### ISSUE TITLE
Feature request: Kavita update cover/metadata, libraries update

### ISSUE THREAD

**Issue by Makhuta** [C9Glax/tranga#234]
## Library scan
First of when the tranga downloads chapter it will try to update all libraries but if one scan is already running it can spam Kavita alerts (this isn't huge deal, the alert is that the scan is being delayed) it would be nice to be able when adding/setting up the Kavita to be able to select the libraries (parsed from provided url after succesful test or something like that) and to run the library update after some configurable timeout or after every job was done or after some number of downloaded chapters.

## Update series cover and metadata
Since the Kavita is not using the series.json and cover.jpg files like Komga it would be nice to have some sort of option to "automatically" change the series cover and metadata, through its API it is possible to get at least most of the "locking" states for each item (for example summary, genres, status, ect.) and it is possible to update each of them through its API.

**Comment by C9Glax** [comment]
> Library scan
First of when the tranga downloads chapter it will try to update all libraries but if one scan is already running it can spam Kavita alerts (this isn't huge deal, the alert is that the scan is being delayed) it would be nice to be able when adding/setting up the Kavita to be able to select the libraries (parsed from provided url after succesful test or something like that) and to run the library update after some configurable timeout or after every job was done or after some number of downloaded chapters.


I have added a Buffer/Timeout on both Notifications and Libraries. Libraries will now update only after no further chapters have been downloaded for 2 minutes, at least every 20 minutes however.
Notifications will be stacked, where no new notification is added within 3 minutes, but will definitely send every 60 minutes.

To enable, use `cuttingedge`, start once (to generate new settings.json file) and then edit "settings.json":
```json
"bufferLibraryUpdates": true,
"bufferNotifications": true
```
